script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1.txt

```diff
@@ -51,21 +51,21 @@ Just like yesterday, she must have come to wake me up so that I wouldn't oversle
 "Ah. I'm sorry, it was weird for me to be so startled.
 Good morning, Hisui. Thank you for coming to wake me up."
 <Page15>
-"This is one of my duties, please do not mind me."
+"This is one of my duties, please me no mind."//「それが私の役割ですから、どうか、お気になさらないようお願いします」
 <Page16>
 Hisui solemnly bows.
-...But, as she does so, she suddenly stops moving and then swiftly raises her head.
+...But, as she does so, she suddenly stops moving and swiftly raises her head.//　……と。その途中でピタリと動きを止めた後、スッと頭を上げて、
 <Page17>
 "No, my duties do not involve surprising you.
 ―――I will make every effort not to startle you tomorrow."
 <Page18>
-...For a maid, she has some rather strange traits.
+...For a maid, she has a subtle strangeness about her.//　……微妙におかしなところのあるメイドさんだった。"おかしなところ" is more like "strange traits" but "subtly strange traits" is awkward imo.
```

Comment:
I feel like the word traits here might be worth keeping, since to say she is subtly strange implies that her entire existence is a bit off, rather than her having a few oddball behaviours. A few subtle quirks?

Reply:
Mmm, I think I agree with Ross, even though "being strange" and "having some strange traits" are very close.